static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -0,0 +1,178 @@
+.newsletters-heading {
+    @include fs-header(5);
+    font-size: 40px;
+    color: $neutral-1;
+    padding-top: $gs-baseline;
+
+    @include mq($from: tablet) {
+        font-size: 60px;
+        padding-top: $gs-baseline * 2;
+  }
+}
+
+.newsletter-card__signup {
+    display: inline;
+    margin-right: 2%;
+}
+
+.newsletter-card__example {
+    display: inline;
+}
+
+.newsletter-card__wrapper {
+    position: relative;
+
+    @include mq(leftCol) {
+        margin-left: gs-span(3) + $gs-gutter;
+    }
+}
+
+.newsletter-card {
+    display: inline-block;
+    float: left;
+    margin-bottom: $gs-baseline;
+    background-color: $neutral-6;
+    border-top: 2px solid $news-accent;
+    width: 100%;
+
+    @include mq(tablet) {
+        width: gs-span(4);
```

Comment:
Anytime I see those words I think of the naff pop band (sorry to any MN fans)

![image](https://cloud.githubusercontent.com/assets/6727874/19474939/a5502476-9529-11e6-851a-8029009cb006.png)